src/Extensions/SchemaScaffolderExtension.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace DNADesign\Elemental\Extensions;
+
+use GraphQL\Type\Definition\CustomScalarType;
+use SilverStripe\Core\Extension;
+use SilverStripe\GraphQL\Manager;
+
+class SchemaScaffolderExtension extends Extension
+{
+    public function onBeforeAddToManager(Manager $manager)
+    {
+        $manager->addType(new CustomScalarType([
+            'name' => 'ObjectType',
+            'serialize' => function ($value) {
+                return (object) $value;
+            },
+            'parseValue' => function ($value) {
+                return (array) $value;
+            },
+            'parseLiteral' => function ($ast) {
+                return $ast->value;
+            },
+        ]), 'ObjectType');
```

Comment:
My only concern with this is that it's applied globally to the DataObjectScaffolder. 

Essentially what this is doing is adding a new type, right? We could isolate this by pointing the scaffolding configuration at a `BaseElementTypeCreator` class (extends TypeCreator), which could then add this.

Otherwise, perhaps this needs to be added directly into core...

cc @unclecheese

Reply:
It probably would make more sense to use `TypeCreator` - although I don't think it's going to change the fact it's applied globally. However it probably makes more sense to namespace the type a little unless it will become an addition to admin.